eoscostarica/eosio-spend-explorer
# feat(webapp): load chart data by default

### Load chart data by default

### What does this PR do?

- Resolve #100 
- Resolve #88 
- Resolve #70
- Resolve #106 

### Steps to test

1. Run the project
2. Open in your browser localhost:3000
3. Click on income in the sidebar
4. See income options

#### CheckList
